seed/challenges/01-front-end-development-certification/jquery.json

```diff
@@ -344,10 +344,12 @@
       "title": "Target the same element with multiple jQuery Selectors",
       "description": [
         "Now you know three ways of targeting elements: by type: <code>$(\"button\")</code>, by class: <code>$(\".btn\")</code>, and by id <code>$(\"#target1\")</code>.",
+        "Although it is possbile to add multiple classes in a single <code>.addClass()</code> call, let's add them to the same element in <em>three sperate ways.",
```

Comment:
You're missing the matching `<em>` tag at the end of this line. Which might be why Travis is giving you issues.